sdk/loadtestservice/Azure.Developer.LoadTesting/tests/LoadTestRunClientTest.cs

```diff
@@ -0,0 +1,324 @@
+        }
+
+        [Test]
+        public async Task BeginCreateOrUpdateTestRun()
+        {
+            LoadTestRunClient loadTestRunClient = CreateRunClient();
+            _loadTestAdministrationClient = CreateAdministrationClient();
+            await _testHelper.SetupLoadTestResourceAndTestScriptAsync(_loadTestAdministrationClient, _testId, _fileName);
+
+            TestRunOperation testRunOperation = await loadTestRunClient.BeginTestRunAsync(
+                WaitUntil.Completed, _testRunId, RequestContent.Create(
+                    new
+                    {
+                        testId = _testId,
+                        displayName = "Run created from dotnet testing framework"
+                    }
+                ));
+
+            Assert.IsTrue(testRunOperation.HasValue);
+            Assert.NotNull(testRunOperation.Value);
+            Assert.IsTrue(testRunOperation.HasCompleted);
+
+            await loadTestRunClient.DeleteTestRunAsync(_testRunId);
+            await _testHelper.SetupLoadTestResourceAndTestScriptAsync(_loadTestAdministrationClient, _testId, _fileName);
+            testRunOperation = await loadTestRunClient.BeginTestRunAsync(
+                   WaitUntil.Completed, _testRunId, RequestContent.Create(
+                        new
+                        {
+                            testId = _testId,
+                            displayName = "Run created from dotnet testing framework"
+                        }
+                   ));
+
+            await testRunOperation.WaitForCompletionAsync();
+            Assert.IsTrue(testRunOperation.HasValue);
+            Assert.NotNull(testRunOperation.Value);
+            Assert.IsTrue(testRunOperation.HasCompleted);
+        }
+
+        [Test]
+        public async Task GetTestRun()
+        {
+            LoadTestRunClient loadTestRunClient = CreateRunClient();
+            _loadTestAdministrationClient = CreateAdministrationClient();
+
+            Response response = await loadTestRunClient.GetTestRunAsync(_testRunId);
+            Assert.NotNull(response);
+        }
+
+        [Test]
+        public async Task GetTestRunFile()
+        {
+            LoadTestRunClient loadTestRunClient = CreateRunClient();
+            _loadTestAdministrationClient = CreateAdministrationClient();
+
+            Response response = await loadTestRunClient.GetTestRunFileAsync(_testRunId, _fileName);
+            Assert.NotNull(response);
+        }
+
+        [Test]
+        public async Task ListTestRuns()
+        {
+            LoadTestRunClient loadTestRunClient = CreateRunClient();
+            _loadTestAdministrationClient = CreateAdministrationClient();
+
+            await _testHelper.SetupTestRunWithLoadTestAsync(_loadTestAdministrationClient, _testId, _fileName, loadTestRunClient, _testRunId, waitUntil: WaitUntil.Started);
+            AsyncPageable<BinaryData> reponse = loadTestRunClient.GetTestRunsAsync();
+
+            Assert.NotNull(reponse);
```

Comment:
You haven't actually done anything yet in this test.  You don't have a response - you have an enumerator that hasn't tried enumerating anything.  Your test should do something more like verify the number of items is accurate.

Reply:
will add it with deeper asserts commit